docs/product-spec.md

```diff
@@ -0,0 +1,138 @@
+weeks for sending out ballots prior to election day and 2 weeks for
+tabulation.
+
+
+## VxMail
+
+We propose VxMail, a set of tools to help small counties quickly scale
+up their vote-by-mail capacity.
+
+### Components
+
+VxMail includes:
+* a Ballot Manager to send mail ballots to voters, log receipt of those ballots, and verify signatures.
+* tools to efficiently open envelopes and extract ballots.
+* a Ballot Tabulator to scan and tabulate ballots.
+
+The Ballot Manager works online for connecting with print-and-mail
+services, and potentially to the state's Election Management
+System. The Ballot Scanner is offline for security purposes.
+
+### Printing and mailing ballots
+
+The VxMail Ballot Manager lets an election official:
+
+* upload the election definition
+* upload the voter mailing list
+* proof generated ballots on screen
+* order those ballots printed and mailed, with a 3-day in-USPS-hands SLA
+* track ballot delivery to voters.
+* track ballot return from voters.
+
+All ballot printing, stuffing, and mailing is managed by VotingWorks
+and its partners, while election officials can track that process via
+the Ballot Manager. Voters receives a ballot and return envelope,
+including, as per best practices, a clear location to sign printed on
+the return envelope.
+
+### Verifying voter signatures
+
+When ballots are received, election officials can verify ballot
+envelope signatures using the VxMail Ballot Manager, which includes an
+envelope scanner:
+
+* An export of voter IDs and signatures, provided by the state, is
+  loaded into the VxMail Ballot Manager
+* Ballot envelopes are scanned into the Ballot Manager in small
+  batches. On screen, expected and actual signatures are displayed for
+  confirmation by election officials. If election officials detect any
+  mismatch on screen, they indicate this with a single tap on the
+  corresponding button, and the scanning process pauses while election
+  officials locate and set aside that ballot envelope. VxMail records
+  digitally that this envelope was rejected. Scanning ballot envelopes
+  continues, one small batch at a time.
+* When all ballots have been scanned, the VxMail Ballot Manager
+  produces an export of successful and unsuccessful voter IDs,
+  including digital images for both sides of the corresponding ballot
+  envelopes. These can then be upload into the state voter management
+  system to record that those voters have cast a ballot.
+  
+If the integration with the state's voter IDs and signature files is
+not an option, election officials may instead use their existing
+Election Management System and look up voter records as ballot
+envelopes come in, either by scanning a voter ID barcode that appears
+on the ballot return envelope, or by manually looking up the voter
+record.
+
+### Opening envelopes and extracting ballots
+
+Depending larger counties, VxMail includes an off-the-shelf automatic
```

Comment:
"Depending larger counties" feels like a bad edit. Fix?

Reply:
yep fixing with another batch of edits. thanks!